src/Bloc/BLCursor.class.st

```diff
@@ -0,0 +1,90 @@
+Class {
+	#name : 'BLCursor',
+	#superclass : 'Object',
+	#category : 'Bloc-Utilities - Selection',
+	#package : 'Bloc',
+	#tag : 'Utilities - Selection'
+}
+
+{ #category : 'accessing' }
+BLCursor class >> crosshair [
+
+	self deprecated: 'Use Cursor instead.' transformWith: '`@rcv crosshair' -> 'Cursor crosshair'.
```

Comment:
I think it must be `crossHair` (uppercase H)

Reply:
Great catch thanks